homeassistant/components/schlage/quality_scale.yaml

```diff
@@ -0,0 +1,84 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
```

Comment:
Another comment about the coordinator: In its current form we kinda spawn an executor thread for every lock with every update. Switching between executor and the event loop is quite expensive, so ideally we have 1 executorjob for all the locks. (I think)